Dockerfile

```diff
@@ -15,4 +15,8 @@ RUN systemctl set-default multi-user.target
 
 COPY setup /sbin/
 
-CMD ["/sbin/init"]
+STOPSIGNAL SIGRTMIN+3
+VOLUME ["/sys/fs/cgroup"]
```

Comment:
Is there a reason why you added the `VOLUME` command?

We're still going to have to mount the cgroup hierarchy from the host, so that's what gets mounted at this mount point and AFAIK the volume listed in the Dockerfile will just be ignored.